dll/win32/shell32/folders/CRecycleBin.cpp

```diff
@@ -1018,6 +1018,65 @@ TRASH_TrashFile(LPCWSTR wszPath)
     return DeleteFileToRecycleBin(wszPath);
 }
 
+static void TRASH_PlayEmptyRecycleBinSound()
+{
+    CRegKey regKey;
+    CHeapPtr<WCHAR> pszValue;
+    CHeapPtr<WCHAR> pszSndPath;
+    DWORD dwType, dwSize;
+    LONG lError;
+
+    /* Open sound key */
+    lError = regKey.Open(HKEY_CURRENT_USER,
+                         L"AppEvents\\Schemes\\Apps\\Explorer\\EmptyRecycleBin\\.Current",
+                         KEY_READ);
+    if (lError != ERROR_SUCCESS)
+        return;
+
+    /* Get the value size */
+    lError = regKey.QueryValue(NULL, &dwType, NULL, &dwSize);
+    if (lError != ERROR_SUCCESS)
+        return;
+
+    /* Allocate a buffer for the value */
```

Comment:
Same with comments like this, typically you should add comments like this only if they add information that can not be derived from the code. (e.g. WHY something is done, or any unexpected / hidden functionalities)

Reply:
Removed.